File: stara/2011/c++/main.cpp
#include <fstream>
#include "zad6_a.cpp"
#include "zad6_b.cpp"
using namespace std;

int main()
{
    cout << zad6_a() << endl << endl << zad6_b();

    ofstream result("wyniki.txt");

    result << zad6_a() << endl << endl << zad6_b();
}

File: stara/2011/python/main.py
import zad6_a
import zad6_b
import zad6_c

# zebranie odpowiedzi z podpunktow i zapisanie ich do pliku
result = '\n'.join([zad6_a.answer, zad6_b.answer, zad6_c.answer])
with open('wyniki.txt', mode='w') as f:
    f.write(result)
